src/login/logind-user.c

```diff
@@ -453,11 +423,6 @@ int user_start(User *u) {
                         return r;
         }
 
-        /* Create cgroup */
-        r = user_start_slice(u);
-        if (r < 0)
-                return r;
-
```

Comment:
hmm, we could also push the prop into the slice unit with SetUnitProperties() here. That way things would continue to work mostly... TasksMax after all is one of those props which can be changed after the fact...